ext/gd/gd.stub.php

```diff
@@ -10,59 +10,60 @@ function gd_info(): array {}
+function imagepalettetotruecolor(GdImage $image): bool {}
 
-function imagecolormatch(GdImage $im1, GdImage $im2): bool {}
+function imagecolormatch(GdImage $image1, GdImage $image2): bool {}
 
-function imagesetthickness(GdImage $im, int $thickness): bool {}
+function imagesetthickness(GdImage $image, int $thickness): bool {}
 
-function imagefilledellipse(GdImage $im, int $cx, int $cy, int $w, int $h, int $color): bool {}
+function imagefilledellipse(GdImage $image, int $center_x, int $center_y, int $width, int $height, int $color): bool {}
 
-function imagefilledarc(GdImage $im, int $cx, int $cy, int $w, int $h, int $s, int $e, int $col, int $style): bool {}
+function imagefilledarc(GdImage $image, int $center_x, int $center_y, int $width, int $height, int $start_angle, int $end_angle, int $color, int $style): bool {}
 
-function imagealphablending(GdImage $im, bool $blend): bool {}
+function imagealphablending(GdImage $image, bool $enable): bool {}
 
-function imagesavealpha(GdImage $im, bool $save): bool {}
+function imagesavealpha(GdImage $image, bool $enable): bool {}
 
-function imagelayereffect(GdImage $im, int $effect): bool {}
+function imagelayereffect(GdImage $image, int $effect): bool {}
 
-function imagecolorallocatealpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorallocatealpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecolorresolvealpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorresolvealpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecolorclosestalpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorclosestalpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecolorexactalpha(GdImage $im, int $red, int $green, int $blue, int $alpha): int|false {}
+function imagecolorexactalpha(GdImage $image, int $red, int $green, int $blue, int $alpha): int|false {}
 
-function imagecopyresampled(GdImage $dst_im, GdImage $src_im, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_w, int $dst_h, int $src_w, int $src_h): bool {}
+function imagecopyresampled(GdImage $dst_image, GdImage $src_image, int $dst_x, int $dst_y, int $src_x, int $src_y, int $dst_width, int $dst_height, int $src_width, int $src_height): bool {}
 
 #ifdef PHP_WIN32
 
-function imagegrabwindow(int $handle, int $client_area = 0): GdImage|false {}
+function imagegrabwindow(int $handle, bool $client_area = false): GdImage|false {}
 
 function imagegrabscreen(): GdImage|false {}
 
 #endif
 
-function imagerotate(GdImage $im, float $angle, int $bgdcolor, int $ignoretransparent = 0): GdImage|false {}
+// TODO: $ignore_transparent is ignored???
+function imagerotate(GdImage $image, float $angle, int $background_color, bool $ignore_transparent = false): GdImage|false {}
```

Comment:
I'm personally happy with `$background_color`, but not opposed to `$bg_color`

Reply:
Although.. "fg" doesn't seem a well known abbreviation (IIRC I've never used it), and since it's used separately from any `$background_color`/`$bg_color` parameter, there wouldn't be any direction about its meaning. That's why I'd prefer if you stayed with the longer versions, given this reasoning makes sense. :)